spec/unit/libraries/command_line_tools_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+include MacOS
+
+describe MacOS::CommandLineTools do
+  context 'when provided an available list of software update products' do
+    before do
+      allow_any_instance_of(MacOS::CommandLineTools).to receive(:available_products)
+        .and_return(<<-SOFTWAREUPDATE_LIST
+                    Software Update Tool
+                    Finding available software
+                    Software Update found the following new or updated software:
+                       * Command Line Tools (macOS El Capitan version 10.11) for Xcode-8.2
+                    \tCommand Line Tools (macOS El Capitan version 10.11) for Xcode (8.2), 150374K [recommended]
```

Comment:
Oh, neat. I didn't realize you could use`\t` for a tab in a heredoc!

Reply:
I meant to use `<<~` here actually - the tabs make more sense in that context.